python/cudf/cudf/tests/test_copying.py

```diff
@@ -12,46 +12,6 @@
 from cudf.tests.utils import assert_eq
 
 
-def test_gather_single_col():
-    col = column.as_column(np.arange(100), dtype=np.int32)
-    gather_map = np.array([0, 1, 2, 3, 5, 8, 13, 21], dtype=np.int32)
-
-    device_gather_map = cupy.asarray(gather_map)
-
-    out = libcudf.copying.gather(col, device_gather_map)
```

Comment:
Did you remove these tests because gather is already tested in tests for `take`?

Reply:
More because this is testing internals rather than a public API so we'd have to change it every time the internals change. And yes, --`gather` is already tested in our `indexing`/`take` tests.